pages/packages/_java_package_repository_id.md

```diff
@@ -0,0 +1 @@
+- `org-slug-java-package-repository-name` is the ID of your Java package repository, based on the org slug and Java package repository names separated by a hyphen. The org slug can be obtained from the end of your Buildkite URL, after accessing _Pipelines_ in the global navigation of your organization in Buildkite.
```

Comment:
The value of this doesn't actually matter, should we mention that?

```suggestion
- `org-slug-java-package-repository-name` is the ID of your Java package repository, based on the org slug and Java package repository names separated by a hyphen. The org slug can be obtained from the end of your Buildkite URL, after accessing _Pipelines_ in the global navigation of your organization in Buildkite. This can be any value you like, but must match the ID used in your `pom.xml`, so we suggest following convention.
```

Reply:
I missed committing this suggestion due to another change I'd already committed to this line. However, I had to modify the formatting of this anyway. It's in the docs at the relevant spot anyway.